draft-pauly-ohai-svcb-config.md

```diff
@@ -73,69 +78,76 @@ document.
 
 {::boilerplate bcp14-tagged}
 
-# The ohttp-configs and ohttp-path SvcParamKeys
+# The oblivious SvcParamKey
+
+The "oblivious" SvcParamKey {{iana}} is used to indicate that a service
+described in an SVCB record can act as an oblivious target. Clients
+can issue requests to this service through an oblivious proxy once
+they learn the key configuration to use to encrypt messages to the
+oblivious target.
 
-The "ohttp-configs" SvcParamKey {{iana}} is used to convey one or more
-key configurations that can be used by clients to issue oblivious requests
-to a target server described by the SVCB record.
+Both the presentation and wire format values for the "oblivious"
+parameter MUST be empty.
 
-In wire format, the value of the parameter is one or more `KeyConfig`
-structures {{OHTTP}} concatenated together. In presentation format,
-the value is the same concatenated `KeyConfig` structures encoded
-in Base64 {{!BASE64=RFC4648}}.
+The "oblivious" parameter can be included in the mandatory parameter
```

Comment:
I am not sure why a target would only support oblivious HTTP for DNR/DDR use cases, please elaborate.

Reply:
It could have multiple services listed — one generic DoH for DDR, and an entirely different record for their ODoH server.

Reply:
It would be helpful to mention the above explanation in the DDR section.

Reply:
Added text in this section where we define the oblivious parameter.